integration/specs/Picklist/picklist-9.spec.js

```diff
@@ -0,0 +1,50 @@
+const PagePicklist = require('../../../src/components/Picklist/pageObject');
+const { ESCAPE_KEY, ARROW_DOWN_KEY, ENTER_KEY } = require('../../constants');
+
+const PICKLIST = '#picklist-9';
+
+const addNewBuildings = () => $('#button-icon_add-new-buildings').click();
+
+describe('Picklist with PicklistOption changed dynamically', () => {
+    beforeAll(() => {
+        browser.url('/#!/Picklist/9');
+    });
+    beforeEach(() => {
+        browser.refresh();
+        const component = $(PICKLIST);
+        component.waitForExist();
+    });
+    it('should render the right amount of options when children are changed dynamically', () => {
+        const picklist = new PagePicklist(PICKLIST);
+        picklist.clickInput();
+        picklist.waitUntilOpen();
+        expect(picklist.getOptionsLength()).toBe(2);
+        browser.keys(ESCAPE_KEY);
+        addNewBuildings();
+        picklist.clickInput();
+        picklist.waitUntilOpen();
+        expect(picklist.getOptionsLength()).toBe(5);
+    });
```

Comment:
this test does not test any component functionality, instead, it check the example code, and we don't want to add unnecessary tests,  so remove it